models/task.js

```diff
@@ -30,18 +35,40 @@ export default class Task {
      PREFIX    adms: <http://www.w3.org/ns/adms#>
      INSERT DATA {
         ${sparqlEscapeUri(uri)} a task:Task;
-                                                mu:uuid ${sparqlEscapeString(id)};
-                                                adms:status ${sparqlEscapeUri(TASK_STATUS_CREATED)};
-                                                task:numberOfRetries ${sparqlEscapeInt(0)};
-                                                dct:created ${sparqlEscapeDateTime(created)};
-                                                dct:modified ${sparqlEscapeDateTime(created)};
+                                                mu:uuid ${sparqlEscapeString(
+                                                  id
```

Comment:
I'd argue this is a lot less readable than it was before

Reply:
my bad for running it without looking, prettier in general doesn't touch literal strings, but it seems it got confused